components/automate-ui/src/app/entities/userperms/userperms.effects.ts

```diff
@@ -18,6 +19,8 @@ import {
 } from './userperms.actions';
 import { UserPermsRequests } from './userperms.requests';
 
+const POLLING_INTERVAL_IN_SECONDS = 300; // every 5 minutes
```

Comment:
How about either
```suggestion
const POLLING_INTERVAL_IN_MINUTES = 5;
```
or
```suggestion
const POLLING_INTERVAL_IN_SECONDS = 300;
```
? Informing our readers that 300s = 5m is underestimatinig them 😉